lifecycle-operator/controllers/common/testcommon/utils.go

```diff
@@ -1,15 +1,41 @@
-package common
```

Comment:
here I don't know what should be the best: `testcommon` or `test` since "common" is already part of the import path 🤔 @bacherfl wdyt?

Reply:
I would definitely not vote for common, either test or testcommon, I used testcommon

Reply:
I would also vote for `testcommon` here - the name `test` might imply that this contains actual unit tests, whereas this contains helper functions that are shared across the tests of different packages